docker/Dockerfile

```diff
@@ -10,6 +11,8 @@ ARG NRL_GIT_REF=main
 ADD --keep-git-dir=true https://github.com/NVIDIA-NeMo/RL.git#${NRL_GIT_REF} /
```

Comment:
Is there a way to override this to allow local context? i.e. I want build a custom docker image with code on my local machine that is not yet committed